Makefile

```diff
@@ -213,8 +213,3 @@ prebuilt-binary:
 		ghcr.io/goreleaser/goreleaser-cross:${GOLANG_CROSS_VERSION} \
 		release --clean
 .PHONY: prebuilt-binary
-
-## govulncheck: Check for vulnerabilities in dependencies.
-govulncheck:
-	@go run golang.org/x/vuln/cmd/govulncheck@v1.0.1 ./...
-.PHONY: govulncheck
```

Comment:
[Optional] I find govulncheck useful and valuable in general, so would like to suggest keeping it if possible. Would something like below work? 
Directing the output of govulncheck to a txt file and then excluding the IBC portion of the error, and displaying the remaining part as the result.

Reply:
Something hacky like that could work but we'd also need to modify the exit code of the script so that CI only fails if the output contains anything after the filtering. 

IMO it isn't worth investing time to implement and maintain the hacky solution so I'd rather drop it entirely for now.

Reply:
I see there is a tracking [issue](https://github.com/celestiaorg/celestia-app/issues/3530) for adding govulncheck back when it supports ignoring errors, so, all good!